### PR TITLE
models for apof

### PR DESCRIPTION
**Pull request by matpython**
I know that I have to remove primary keys, just wanted to see if the commit issue is over

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class Comments(models.Model):
+    id_comment = models.AutoField(primary_key=True)
+    content = models.CharField(max_length=255)
+    spice_rate = models.DecimalField(max_digits=5, decimal_places=1)
+    rank = models.DecimalField(max_digits=5, decimal_places=1)
+
+    def __str__(self):
+        return self.name
```
Comment (Orzech150) [inline models.py]: There's no `name` field in this model.

File: models.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class Comments(models.Model):
+    id_comment = models.AutoField(primary_key=True)
+    content = models.CharField(max_length=255)
+    spice_rate = models.DecimalField(max_digits=5, decimal_places=1)
+    rank = models.DecimalField(max_digits=5, decimal_places=1)
+
```
Comment (Orzech150) [inline models.py]: How about `DateTimeField`? 
We could use it in `__str__` method as well as display it on page.

File: apof/restaurant/models.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        return self.name
+
+
+class Comments(models.Model):
+    content = models.CharField(max_length=255)
+    spice_rate = models.DecimalField(max_digits=5, decimal_places=1)
+    rank = models.DecimalField(max_digits=5, decimal_places=1)
+
+    def __str__(self):
+        return self.name
```
Comment (Orzech150) [inline apof/restaurant/models.py]: There's no `self.name` in this model.

Comment (kamilgrela) [inline apof/restaurant/models.py]: Ingridients.meal is redundant.
I propose removing ingridients.meal and adding many to many field in Meal model.

**Comment by pkucmus** [comment]
Closing due to inactivity.

**Pull request closed without merging**